syntax = "proto3";

package chat;

message ChatRequest
{
	int32		from_uid	= 1;  // from_uid
	int32		to_uid		= 2;  // to_uid
	string		msg			= 3;  // msg
}

message ChatReply
{
	int32		from_uid	= 1;  // from_uid
	int32		to_uid		= 2;  // to_uid
	string		msg			= 3;  // msg
}

message Person {
  int32 age = 1;
  string name = 2;
}

message Family {
  repeated Person person = 1;
} 